Google-Summer-of-Code-2025.md

```diff
@@ -43,6 +43,18 @@ Furthermore, note that GSoC isn't just about code -- it's also about communicati
 You might also enjoy the "weekly journals" from some of our previous contributors: **[Rd4dev](https://medium.com/@rd4dev)** and **[@theMr17](https://medium.com/@Mr_17)**.
 
 
+## Selected Projects
+
+The following projects (with linked proposals) are selected as part of GSoC 2025:
+
+* [Mohit](pdfs/GSoC25MohitRuwatia.pdf): Clean up the structure for study guides and worked examples. Mentor: Kevin.
```

Comment:
Let's title the proposals consistently -- include first name and surname for all.